GRAPHICS/DoesLatestBriefStringExist.md

```diff
@@ -0,0 +1,33 @@
+---
+ns: GRAPHICS
```

Comment:
You're missing the aliases, which should include the previous native name(s), you can see [here](https://github.com/citizenfx/natives/edit/master/NETWORK/NetworkUseHighPrecisionBlending.md) for an example, though note you don't have to include the 'Old Name' tidbit it's just leftovers from a NativeDB import